segmentation/utils/visualize_test.py

```diff
@@ -10,3 +16,26 @@ def test_visualize_z_scores():
     # Assign random phenotype titles
     pheno_titles = [chr(i) for i in range(ord('a'), ord('z') + 1)]
     plot = visualize.visualize_z_scores(z, pheno_titles)
+
+
+def test_get_sort_data():
+    rand_type = random.choices(string.ascii_lowercase, k=100)
+    ids = random.choices(range(1, 10), k=100)
+    random_data = pd.DataFrame.from_dict({"PatientID": ids, "cell_type": rand_type})
+    print("Before sorting")
+    print(random_data)
+    print("----------------------------------")
+    print("After sorting")
+    print(visualize.get_sorted_data(random_data, "PatientID", "cell_type"))
+
+
+def test_visualize_cells():
+    rand_type = random.choices(string.ascii_lowercase, k=100)
+    ids = random.choices(range(1, 3), k=100)
+    df = pd.DataFrame.from_dict({"PatientID": ids, "cell_type": rand_type})
+    visualize.visualize_patient_population_distribution(df, "PatientID", "cell_type", save_dir="")
+
+    # Check if correct plots are saved
+    assert path.exists("PopulationDistribution.png"), "Should be showing distribution"
+    assert path.exists("TotalPopulationDistribution.png"), "Should be showing total count"
+    assert path.exists("PopulationProportion.png"), "Should be showing proportion"
```

Comment:
Remove the side comments or else change to asterisk notation (ex. # should be showing distribution)